{"name": "Ошибка '409' при создании двух одинаковых курьеров", "status": "passed", "steps": [{"name": "Создаём двух одинаковых курьеров", "status": "passed", "start": 1747762666367, "stop": 1747762666682}], "start": 1747762666367, "stop": 1747762666683, "uuid": "64b1ca68-7995-4d6b-9912-db2cdc7caf8a", "historyId": "acd8d8e8195c89825fb471e3fc9b39be", "testCaseId": "acd8d8e8195c89825fb471e3fc9b39be", "fullName": "tests.test_creating_courier.TestCreatingCourier#test_error_when_creating_two_identical_couriers", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_creating_courier"}, {"name": "subSuite", "value": "TestCreatingCourier"}, {"name": "host", "value": "KOMPUTER"}, {"name": "thread", "value": "7720-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_creating_courier"}]}